python/py-dulwich/Portfile

```diff
@@ -18,26 +18,30 @@ long_description    Simple Pure-Python implementation of the Git file \
 
 homepage            https://www.dulwich.io
 
-checksums           rmd160  c81b9d7f4b6a65b4699b4f2e046fe471aa4dc29f \
-                    sha256  08b7e0ad62c06c82810ac91925512d2a6ae9875625fc67e8d50150b089bd0a94 \
-                    size    363564
+checksums           rmd160  3557712ed0c45d73c104a5dd5dc215bfb3e940cb \
+                    sha256  2b2512ad80e03a1d9ffaafc38d884992b40ac7cfa6ddd814db098733365712dc \
+                    size    370540
 
-python.versions     27
+python.versions     27 38
 
 if {${name} ne ${subport}} {
-    conflicts           py${python.version}-dulwich-devel
-    if {[string match "*-devel" $subport]} {
-        conflicts           py${python.version}-dulwich
+    # 0.20 dropped support for Python 2.7
+    if {[string match "py27-*" ${subport}]} {
```

Comment:
to me ```if {${python.version} == 27} {``` is more readable and that's used in many other ports, so I would suggest doing that here as well

Reply:
Fixed!